app/components/facilities-section.js

```diff
@@ -1,5 +1,6 @@
 import Ember from 'ember'; // eslint-disable-line
 import mapboxgl from 'mapbox-gl'; // eslint-disable-line
+import geoViewport from 'npm:@mapbox/geo-viewport'; // eslint-disable-line
```

Comment:
Excellent - but this dependency wasn't saved to package.json

Reply:
Fixed.